servant-server/src/Servant/Server.hs

```diff
@@ -17,6 +17,8 @@ module Servant.Server
   , -- * Handlers for all standard combinators
     HasServer(..)
   , Server
+  , EmptyAPIServer
```

Comment:
I'd name these just
```haskell
EmptyServer
emptyServer
emptyClient
EmptyClient
```

Not sure whether `EmptyAPI` should also be just `Empty`, we already have clash-y names in api-type vocabulary, so I'm :+1: to dropping `API` from the names.